environment-docker.yml

```diff
@@ -2,57 +2,63 @@ name: waisn-tech-tools
 channels:
```

Comment:
I wrote a guide on how to update this file. See the **Docker** section:

* https://github.com/ErrorsAndGlitches/waisn-tech-tools/tree/2-sns-integration#updating-the-environment

There are still a lot more changes in this file than I would expect to see.

Reply:
Working on this